pkg/apis/multishare/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package multishare
+
+// GroupName is the group name used in this package
+const (
+	GroupName = "multishare.filestore.csi.storage.gke.io"
```

Comment:
I know this is multishare specific crd but can we use `filestore.csi.storage.gke.io` ?

Reply:
I think it's beneficial to have some clarity from `filestore.csi.storage.gke.io` which is the driver it self?